examples/ex30p_proposed.cpp

```diff
@@ -0,0 +1,532 @@
+//                       MFEM Example 30 - Parallel Version
+//
+// Compile with: make ex30p
+//
+// Sample runs:  mpirun -np 4 ex30p -m ../data/inline-segment.mesh
```

Comment:
`❯ ./ex30p_proposed` runs fine. However, when I execute the first suggested sample run, `❯ ./ex30p_proposed -m ../data/inline-segment.mesh`, I receive a segmentation fault. The segmentation also appears when running `❯ mpirun -np 4 ex30p_proposed -m ../data/inline-segment.mesh`. I have not checked the remaining sample runs in this file.

Reply:
Actually I had to tweak HYPRE's implementation of AMS so that it would support 1D problems. I had forgotten about this. I have a pull request ([#269](https://github.com/hypre-space/hypre/pull/269)) with HYPRE which I should really followup on.

Reply:
Nice catch, I didn't try this sample run. @mlstowell should this sample run be removed until the hypre PR is merged, since otherwise the MFEM testing will fail?

Reply:
I have removed the 1D sample runs.